libs/ui/elements/src/lib/markdown-editor/markdown-editor.component.html

```diff
@@ -0,0 +1,38 @@
+<div class="h-full flex flex-col">
+  <div class="flex-none w-full flex flex-row items-center negative-margin">
```

Comment:
Note that you can use explicit TW classes here: `-mt-[1rem] -mb-[0.75rem]` instead of using separate CSS